framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -465,6 +477,9 @@ def set_stimulus(self, **data):
                                                           is_simulator_load, self.last_loaded_form_url,
                                                           cherrypy.request.method, is_model_fragment=True,
                                                           is_surface_simulation=session_stored_simulator.is_surface_simulation)
+        is_branch = common.get_from_session(common.KEY_IS_SIMULATOR_BRANCH)
+        if is_branch:
+            rendering_rules.enable_branch_class = "enable_branch"
```

Comment:
Why do this if block everywhere? You could keep 'enable_branch' class directly in the html template and add it to the fieldset only when rendering_rules.is_branch is true